packages/x-tree-view/src/internals/plugins/useTreeViewFocus/useTreeViewFocus.ts

```diff
@@ -120,7 +154,6 @@ export const useTreeViewFocus: TreeViewPlugin<UseTreeViewFocusSignature> = ({
   return {
     getRootProps: (otherHandlers) => ({
       onFocus: createHandleFocus(otherHandlers),
-      onBlur: createHandleBlur(otherHandlers),
       'aria-activedescendant': activeDescendant ?? undefined,
```

Comment:
Nitpick: It doesn't seem that https://www.w3.org/WAI/ARIA/apg/patterns/treeview/examples/treeview-1a/ uses it, nor does https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-activedescendant#associated_roles list `tree` as one of the roles that this attribute has any effect. 🤷 
Have you checked/considered if it is needed? 🤔

Reply:
Could you please expand, I'm not sure to follow you here

Reply:
It's not a topic related to this PR, but a general thing I noticed - it looks like this field is redundant and not used in case of `tree` `role`. 🤔

Reply:
`aria-activedescendant` or `onBlur`?

Reply:
`aria-activedescendant` 😉

Reply:
That's interesting
If, by any change, this attribute is pointless, we would no longer need to store the idem id DOM attribute at the tree level.
But I would be cautious with this change and double check before.

Maybe creating an issue would be worth it :+1:

Reply:
I've created an issue for it: https://github.com/mui/mui-x/issues/12494 😉